src/index.ts

```diff
@@ -6,16 +6,25 @@ import 'pixi-sound';
 import 'pixi-animate';
 import TemplateGame from './TemplateGame';
 import * as Transition from './assets/Transition';
+import { CONFIG } from './config/config';
```

Comment:
Just noting that the config.ts file is generated in the build process, so it's not in the template repo.